runtime/onert/core/src/ir/pass/ConstantInsertionPass.cc

```diff
@@ -35,9 +35,9 @@ void ConstantInsertionPass::callback(const OperationIndex &node_index, Operation
   const auto layout = op_seq_lower_info->layout();
   const auto factor = operand::PermuteFactor{backend, layout};
 
-  for (auto it = node.getInputs().mbegin(); it != node.getInputs().mend(); ++it)
+  std::unordered_set<OperandIndex> unique_inputs(node.getInputs().begin(), node.getInputs().end());
```

Comment:
@ragmani I prefer my solution. Changed code requires `unordered_set` and based on some assertion the below and not intuitive.

Reply:
I think origin way makes duplicated calling for the same inputs. I don't think this is not intuitive. 
And what about assertion?